src/otx/v2/adapters/torch/lightning/anomalib/engine.py

```diff
@@ -113,6 +117,15 @@ def _update_callbacks(
             ),
         ]
 
+    def _load_checkpoint(
```

Comment:
@ashwinvaidya17 @samet-akcay 
I have a question about the load_checkpoint part.
I wrote a function called adapters/torch/lightning/engine.py::LightningEngine::load_checkpoint.
<img width="354" alt="image" src="https://github.com/openvinotoolkit/training_extensions/assets/38045080/c529ca52-04e6-4f20-83cc-13f6e88ad4a5">

However, I realized that this is a problem when predicting or exporting from anomalib Module.
I want this function to load the model when a checkpoint comes in as input, but do I need to do something different?